scripts/testing/setup-mock-test-env.py

```diff
@@ -370,7 +370,9 @@ def install_pip_packages_to_mock(mock_command, packages):
     cmd = _prepare_command(mock_command)
 
     cmd = _run_cmd_in_chroot(cmd)
-    cmd.append('pip3 install --install-option="--install-scripts=/usr/bin" {}'.format(packages))
+    cmd.append(
+        'python3 -m pip install --install-option="--install-scripts=/usr/bin" {}'.format(packages)
```

Comment:
Will this work on RHEL? IIRC there is platform-python, python3.6 and such, and you use alternatives to set up what is python2, python3, python etc.

Reply:
`python3` should call python3.6 on RHEL 8, so it should be fine.

Reply:
platform-python is pre-installed but we are dragging in "user" python.

Reply:
It would be interesting to switch but I don't think it's really worth the effort.